WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -113,28 +119,69 @@ public View onCreateView(final LayoutInflater inflater, ViewGroup container, Bun
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
 
-        mUsernamesContainer = (ViewGroup) getView().findViewById(R.id.usernames);
+        mUsernamesContainer = (ViewGroup) view.findViewById(R.id.usernames);
+        mUsernamesContainer.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                EditTextUtils.showSoftInput(mUsernameEditText);
+            }
+        });
 
         String role = mRole;
         if (role == null) {
             role = loadDefaultRole();
         }
 
-        mUsernameEditText = (EditText) getView().findViewById(R.id.invite_usernames);
+        mUsernameEditText = (MultiUsernameEditText) view.findViewById(R.id.invite_usernames);
+        mUsernameEditText.setText(" ");
+        mUsernameEditText.setSelection(1);
+
+        mUsernameEditText.setOnSelectionChangeListener(new MultiUsernameEditText.OnSelectionChangeListener() {
+            @Override
+            public void onSelectionChanged(int selStart, int selEnd) {
+                if (selEnd == 0) {
```

Comment:
I'm afraid this picks up a case we might not want the deletion to happen: when doing a "select all" in the EditText the empty space is selected as well and so, deleting it ends up removing the previous username as well.